x-pack/plugins/security/server/authentication/providers/oidc.ts

```diff
@@ -248,14 +248,18 @@ export class OIDCAuthenticationProvider extends BaseAuthenticationProvider {
     try {
       // This operation should be performed on behalf of the user with a privilege that normal
       // user usually doesn't have `cluster:admin/xpack/security/oidc/authenticate`.
-      result = await this.options.client.callAsInternalUser('shield.oidcAuthenticate', {
-        body: {
-          state: stateOIDCState,
-          nonce: stateNonce,
-          redirect_uri: authenticationResponseURI,
-          realm: this.realm,
-        },
-      });
+      result = (
```

Comment:
**nit**: for the endpoints where we need to use `transport.request`, can we add a comment linking to https://github.com/elastic/elasticsearch/issues/67189? That will be really helpful for future me to remember why we did it this way 🙂

Reply:
Yep, will do, thanks!